doc/source/releases/18.01_announce.rst

```diff
@@ -14,9 +14,7 @@ Highlights
 
 **Web Server and Configuration**
   The default web server used by Galaxy has changed from Paste to `uWSGI <https://uwsgi-docs.readthedocs.io/>`__ and the default configuration file for Galaxy is now ``config/galaxy.yml`` instead of ``config/galaxy.ini``.
-  uWSGI is more production ready and allows Galaxy to scale better in its default
-  configuration. In the future uWSGI will allow Galaxy to setup GIE proxies without additional
-  configuration and use modern web technologies such as web sockets.
```

Comment:
I think it would be better to keep here at least the first sentence (`uWSGI is more production ready and allows Galaxy to scale better in its default configuration.`), to explain why we are making this change.

Reply:
re-added